boards/xtensa/nxp_adsp_imx8ulp/nxp_adsp_imx8ulp_defconfig

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_NXP_IMX8ULP=y
+CONFIG_SOC_NXP_IMX8ULP=y
+CONFIG_BOARD_NXP_ADSP_IMX8ULP=y
+
+CONFIG_GEN_ISR_TABLES=y
+CONFIG_GEN_IRQ_VECTOR_TABLE=n
+
+CONFIG_XTENSA_RESET_VECTOR=y
```

Comment:
Why's this needed here? You're already enabling this via CONFG_SOC_SERIES_NXP_IMX8ULP